### ISSUE TITLE
"none" option no available on email configuration

### ISSUE THREAD

**Issue by fantapereira** [nextcloud/server#6304]
## Steps to reproduce
1. set email parameters
2. send a test mail
3. confirm email is working

## Expected behaviour
configure email without authentication.

## Actual behaviour
i have a production owncloud and a test Nextcloud installed manually, both have "none" option in authenticate box. NC does not have that option available. i have "login, plane, NT login".

## Server configuration

**Server version:**12.0.2

**VM version** (master / released version)

**Updated from an older installed version or fresh install:**

## Network
**Do you use DHCP?**

**Is port 80 and/or 443 open?**

#### Log file (/var/ncdata/nextcloud.log)
```
Ocurrió un problema al enviar el mensaje de correo electrónico. Revise su configuración. (Error: Expected response code 250 but got code "530", with message "530 5.7.1 Client was not authenticated ")
```

#### Installation log (command output)
```
Insert the CLI output here
```

**Comment by fantapereira** [comment]
anyone can assist?

**Comment by Harbardr** [comment]
Hi @fantapereira 

The problem is with the line 99 on the “/var/www/nextcloud/settings/templates/settings/admin/additional-mail.php” file.

Replace the simple quote by a double quote.

`<select name="mail_smtpauthtype" id="mail_smtpauthtype'>`

by

`<select name="mail_smtpauthtype" id="mail_smtpauthtype">`

KR,

**Comment by MorrisJobke** [comment]
Fix is in #11416
